APCAppCore/APCAppCore/UI/Onboarding/APCStudyOverviewCollectionViewController.m

```diff
@@ -238,7 +238,7 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
         
         NSString *filePath = [[NSBundle mainBundle] pathForResource: studyDetails.detailText ofType:@"html" inDirectory:@"HTMLContent"];
         NSAssert(filePath, @"Expecting file \"%@.html\" to be present in the \"HTMLContent\" directory, but didn't find it", studyDetails.detailText);
-        NSURL *targetURL = [NSURL URLWithString:filePath];
```

Comment:
is correct, if we use fileURLWithPath: do not need stringByAddingPercentEscapesUsingEncoding :, only fileURLWithPath already does everything and I did not know =)